### ISSUE TITLE
Installer finds false positive controller import in esbuild setup

### ISSUE THREAD

**Issue by julianrubisch** [stimulusreflex/stimulus_reflex#583]
Just found out that this line:

https://github.com/stimulusreflex/stimulus_reflex/blob/165c2fae77b164e2f82306f4c692f97040b9c7fd/lib/tasks/stimulus_reflex/install.rake#L41

fails in a new `jsbundling-esbuild` setup because the following is typically present:

```js
import controllers from './**/*_controller.js'
controllers.forEach(controller => {
  application.register(controller.name, controller.module.default)
})
```

thought I'd make a quick note of it.

**Comment by marcoroth** [comment]
This should be resolved via #610

**Comment by marcoroth** [comment]
Resolved via #610
